tests/webapp/api/test_performance_data_api.py

```diff
@@ -870,6 +872,7 @@ def test_perfcompare_results_with_only_one_run(
             'new_stddev': round(statistics.stdev(sig2_values), 2) if len(sig2_values) >= 2 else 0,
             'base_stddev_pct': 0,
             'new_stddev_pct': 0,
+            'graphs_link': 'graphs?highlightedRevisions=08038e535f58&highlightedRevisions=1377267c6dc1series=test_treeherder_jobs%2C2%2C1%2C1&timeRange=172800',
         },
     ]
```

Comment:
how can I make a test for different repository comparison data 
@beatrice-acasandrei

Reply:
You can make a new test and create a different repository to use.